README.md

```diff
@@ -15,6 +15,7 @@ White hat hackers can [return funds](https://twitter.com/nomadxyz_/status/155529
   - `data/hack/`: transactions from the hack & events emitted within them
   - `data/nomad/`: lifecycle events of Nomad messages, used to identify transactions from the hack
   - `data/recovery/`: ERC20 transfer events and token amounts sent into the Official Nomad Recovery Address
+  - `white_hats_as_of_<date>.json`: all white hat addresses as identified by TRM Labs
```

Comment:
minor, should be `data/white_hats_as_of_<date>.json` to follow the format of the other folders. Or just remove data from all of them.